app/helpers/application_helper/button/miq_request_reload.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationHelper::Button::MiqRequestReload < ApplicationHelper::Button::MiqRequest
+  needs :@showtype, :@record, :@request_tab
+
+  def visible?
+    return false unless super
+    true
+  end
```

Comment:
```
false unless super
true
```
that is equal to `super` right?

and if there's just super then we don't need the method

and if that is the case we don't need the class at all and cau use the superclass in the toolbar definition instead